code/modules/mob/living/simple_animal/hostile/true_changeling.dm

```diff
@@ -0,0 +1,190 @@
+#define TRUE_CHANGELING_REFORM_THRESHOLD 0 //Can turn back at will, by default
+#define TRUE_CHANGELING_PASSIVE_HEAL 3 //Amount of brute damage restored per tick
+#define TRUE_CHANGELING_FORCED_REFORM 180 //3 minutes
+
+//Changelings in their true form.
+//Massive health and damage, but move slowly.
+
+/mob/living/simple_animal/hostile/true_changeling
+	name = "true changeling"
+	real_name = "true changeling"
+	desc = "Holy shit, what the fuck is that thing?!"
+	speak_emote = list("says with one of its faces")
+	emote_hear = list("says with one of its faces")
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "horror1"
+	icon_living = "horror1"
+	icon_dead = "horror_dead"
+	speed = 1
+	gender = NEUTER
+	a_intent = "harm"
+	stop_automated_movement = 1
+	status_flags = CANPUSH
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	health = 250
+	maxHealth = 250 //pretty durable
+	force_threshold = 10
+	healable = 0
+	environment_smash = 1 //Tables, closets, etc.
+	melee_damage_lower = 35
+	melee_damage_upper = 35
+	see_in_dark = 8
+	see_invisible = SEE_INVISIBLE_MINIMUM
+	wander = 0
+	attacktext = "tears into"
+	attack_sound = 'sound/creatures/hit3.ogg'
```

Comment:
I'm not sure what sound this is but I think the slaughter demon attack noise might fit better considering what its attacktext is

Reply:
it fits, trust me. Floyd approved.

besides every single big monster uses slaughter demon sounds can we take a break from that